modules/core/src/main/java/org/apache/ignite/internal/processors/query/schema/management/SchemaManager.java

```diff
@@ -1086,6 +1087,20 @@ public Collection<TableDescriptor> tablesForCache(String cacheName) {
         }
     }
 
+    /**
+     * @param schemaNamePtrn Filter by schema name. Can be {@code null} to don't use the filter.
+     * @param tblNamePtrn Filter by table name. Can be {@code null} to don't use the filter.
+     *
+     * @return Collection of index descriptors filtered by a given patterns.
+     */
+    public Collection<IndexDescriptor> indexes(String schemaNamePtrn, String tblNamePtrn) {
```

Comment:
Let's filter indexes by patters in `JdbcMetadataInfo`, not here. This logic is specific for jdbc connector.